### ISSUE TITLE
attachments in a long email conversation

### ISSUE THREAD

**Issue by solideo1971** [k9mail/k-9#1979]
### Expected behavior
Tell us what should happen

In the previous versions before the 5.201 if we receive a messagge with 25 replies, and an attachment the same can be showed clicking on the button on the right part of top of the message.

### Actual behavior
Tell us what happens instead

if we received the same messages now (25 replies), the attachement is showed only after scroll all messages, at the end. Is not so easy and immediate to see.

### Steps to reproduce
1.
2.
3.

### Environment
K-9 Mail version:
5.201 - 5.202
Android version:
6.0 - 6.0.1
Account type (IMAP, POP3, WebDAV/Exchange):
IMA - POP3

**Comment by philipwhiuk** [comment]
Duplicate of #1977
